### PR TITLE
Feature/103 Download single file style

### PR DESCRIPTION
**Pull request by kcinay055679**

### CODE REVIEW

**Review by RandomTannenbaum — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/fragments/exam-file.html [hunk 1]
```diff
@@ -1 +1,8 @@
-<a th:href="@{|/exams/download/${examFile.getSubjectName}/${examNumberForm.getExamNumber()}|}" th:text="${examFile.getSubjectName()}" class="link-primary"></a>
\ No newline at end of file
+
+
+
+
```
[the remark this answers is not in the document]
Comment (Miguel7373) [reply src/main/resources/templates/fragments/exam-file.html]: Done

**Review by RandomTannenbaum — DISMISSED** [review]

File: src/main/resources/templates/search.html [hunk 2]
```diff
@@ -11,15 +11,22 @@
     <body>
         <section layout:fragment="content" class="content">
             <span class="display-4">Suche nach einer Prüfung...</span>
-
             <div th:replace="~{fragments/exam-number-input}"></div>
-            <div class="exam-files" th:each="examFile: ${examFiles}">
+            <h3
+                th:if="${examFiles != null && examFiles.size() >= 2}"
```
Comment (RandomTannenbaum) [inline src/main/resources/templates/search.html:16]:
```suggestion
                th:if="${examFiles != null && examFiles.size() > 0}"
```

**Review by RandomTannenbaum — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/search.html (hunk 2 above)
Comment (ManuelMoeri) [reply src/main/resources/templates/search.html:16]: :+1:

**Review by RandomTannenbaum — DISMISSED** [review]

**Review by MasterEvarior — CHANGES_REQUESTED** [review]
The code looks good just two things that I've spotted:
- Everything except the subject title (i.e. "Öffentliches Recht") is center aligned. I think it would look better if this is also aligned the same way, which would then also be consistent with the mockup.
- If the subject is very long it gets cut off to the right, which I think this is fine. The user however has no possibility to discerne between to long subject which start the same (i.e. "Öffentliches Recht der Fakultät der modernen Sprachen" and "Öffentliches Recht der Fakultät der modernen Mathematik"). I think the user should be able to tell the difference between the two. Maybe a tooltip would help?
image

**Review by MasterEvarior — APPROVED** [review]
LGTM :+1:

**Pull request merged**